ghpages.mk

```diff
@@ -68,9 +71,21 @@ $(GHPAGES_TMP): fetch-ghpages
 
 PUBLISHED := index.html $(drafts_html) $(drafts_txt)
 
+ifeq (master,$(SOURCE_BRANCH))
+ifeq (true,$(PUSH_GHPAGES))
+EXTANT_BRANCHES := $(subst refs/heads/,,$(foreach remote,$(shell git remote),$(shell git ls-remote --heads $(remote) | cut -f 2)))
```

Comment:
Ugh, that's a double-reason not to use this: it will block a regular `make` on the network.

I have no problem with having this not use remotes that weren't recently fetched.  We already check if the directory is old, so the only risk is that the remote has a big gap.  On CI, we don't have any other remotes, so it's going to prune aggressively there anyway.